tensorflow_addons/image/transform_ops.py

```diff
@@ -22,9 +22,8 @@
 from tensorflow.python.framework import ops
 from tensorflow_addons.utils.resource_loader import get_path_to_datafile
 from tensorflow.python.framework import dtypes
-from tensorflow.python.ops import array_ops
-from tensorflow.python.ops import math_ops
-
+from tensorflow.python import array_ops
```

Comment:
Apologies for my misleading message. Use `tf.*` public api like what we did in https://github.com/tensorflow/addons/pull/55 .

Reply:
As said before, please use public api of tf, say `tf.constant`, rather than `array_ops.constant`. Please refer to #55